libraries/botbuilder-dialogs-adaptive/src/actions/httpRequest.ts

```diff
@@ -318,6 +323,12 @@ export class HttpRequest<O extends object = {}> extends Dialog<O> implements Htt
                         headers: instanceHeaders,
                     });
                     break;
+                case HttpMethod.HEAD:
```

Comment:
To avoid repeating code, HEAD could be added to the DELETE and GET case.

Reply:
Thanks for the feedback. Resolving as mentioned.